github/activity.go

```diff
@@ -12,3 +12,59 @@ package github
 type ActivityService struct {
 	client *Client
 }
+
+// FeedLink represents a HAL-style link.
```

Comment:
sorry to nitpick here :grin:

This is really only a "HAL-style link" when it is serialized as part of a "_links" JSON object.  Which this is when it's part of a `Feeds.Links`.  But for the purposes of documenting what the type itself is, outside the context of how it may be used in some cases, it's more accurate to simply say that it's a "link to a related resource".

An interesting spec to read on this is [rfc5988](https://tools.ietf.org/html/rfc5988) which defines links outside of how they may be serialized in section 3.  It does then go own to define the Link HTTP header in section 5, but that's really because it existed in a previous version of HTTP and got dropped because there wasn't a solid use case for it at the time.